pkg/aws_config_server/webserver.go

```diff
@@ -181,5 +182,6 @@ func GetRouter(
 
 	loggingHandler := handlers.CombinedLoggingHandler(os.Stdout, router)
 	recoveryHandler := handlers.RecoveryHandler()(loggingHandler)
-	return recoveryHandler
+	honeycombHandler := hnynethttp.WrapHandler(recoveryHandler)
```

Comment:
```suggestion
       if <honeycomb enabled> {
	   handler = hnynethttp.WrapHandler(handler)
       }
```
Instead of putting this one as the "outtermost handler" should we move it up a couple lines so it is processed after the logging and recovery handlers?